src/lib/logger.ts

```diff
@@ -113,8 +113,14 @@ export const logInboundMessage = (message: GleeMessage) => {
 }
 
 export const logOutboundMessage = (message: GleeMessage) => {
+  let verb: string
+  const event = message.eventNames()[0]
   const icon = message.broadcast ? '⇶' : '↗'
-  const verb = message.broadcast ? 'broadcasted' : 'sent'
+  if (message.serverName === 'websockets' && event !== 'reply') {
```

Comment:
You can't do `message.serverName === 'websockets'` here because `websockets` is a name the user gave to this server but it doesn't have to be like this. We may call a websockets server `production` or `testing` or even other languages like Spanish or French. So we can't rely on this.

Reply:
> However, when we're sending to a WebSockets server, it should say "was broadcasted" unless it was a reply to a specific WebSockets client.

Does this mean logs should have `was broadcasted` for all bidirectional protocols?

Reply:
Actually no. MQTT is also bidirectional and it shouldn't say `was broadcasted`. I think it should rely on the fact that it's a server and not a client. So far, when we "see" `ws` on the spec file, we create a WebSockets **server**. But when we see `mqtt` we create an MQTT **client**. So, I don't think this can be solved before https://github.com/asyncapi/glee/issues/259.

Reply:
Then I will wait for https://github.com/asyncapi/glee/issues/259 for this one.